velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1096,6 +1098,29 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct CurrentTimeFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  const date::time_zone* sessionTimeZone_ = nullptr;
```

Comment:
This local variable should not end in "_". Please remove it.

Reply:
Done